bootstrap/pkg/kfapp/gcp/gcp.go

```diff
@@ -77,6 +77,14 @@ const (
 // The namespace for Istio
 const IstioNamespace = "istio-system"
 
+var DEFAULT_EXP_BACKOFF = backoff.ExponentialBackOff{
```

Comment:
Can we have a function like NewKubeflowBackOff() which returns a new backoff object each time?
Otherwise we need to reset it each time before we use it.
Also if we use backoff in multi-thread case, they might overwrite each other (on same backoff object).

Reply:
done.